railties/test/commands/credentials_test.rb

```diff
@@ -76,7 +86,7 @@ class Rails::Command::CredentialsCommandTest < ActiveSupport::TestCase
   private
     def run_edit_command(editor: "cat")
       switch_env("EDITOR", editor) do
-        rails "credentials:edit"
+        rails "credentials:edit", allow_failure: true
```

Comment:
Too lenient. Only one instance of the command is expected to fail. Let's pass it in as a keyword argument from your test.